lib/gateways/models/createResourceFromModel.test.js

```diff
@@ -22,6 +22,18 @@ describe('createResourceFromModel', () => {
       }
     };
 
+    expect(createResourceFromModel(record)).toMatchSnapshot();
+  });
+  it('maps properties from an Airtable record with prompts', () => {
+    const record = {
+      id: 'rec1hMVCuHzXQ2Bk0',
+      fields: {
+        supportingInformation: 'It’s important to look after your health as not seeking support for your health condition could be extremely dangerous.',
+        prompts: 'How is your health at the moment?',
+        promptTags: ['Health']
+      }
```

Comment:
I'm confused about where this data is coming from. It looks like we didn't add any test setup with it in, in this PR - is it from the real airtable?